frontend/src/components/ViewActivities/activity.js

```diff
@@ -1,25 +1,106 @@
 import React from 'react';
-import Card from 'react-bootstrap/Card';
 import * as time from '../Utils/time.js';
 import * as DB from '../../constants/database.js'
 import '../../styles/activities.css';
-import Accordion from 'react-bootstrap/Accordion';
+import { getField } from './activityfns.js';
+import { Accordion, Button, Card, Col, Form, Row } from 'react-bootstrap';
 
+/**
+ * Returns a dropdown of all the timezones.
+ */
+function timezonePicker() {
+  // TODO: Make this dropdown. (#51)
+  return <div></div>
+}
+
+function submitEdit(k) {
+  console.log(k);
+}
```

Comment:
Where else is this used?

Reply:
That function is not supposed to exist. oops.